docs/k8s-cluster/kubeflow.md

```diff
@@ -104,14 +104,14 @@ Verify Ceph is running and/or a DefaultStorageClass is defined:
 
 ```
 kubectl get storageclass | grep default
-./scripts/ceph_poll.sh
+./scripts/k8s/ceph_poll.sh
 ```
 > NOTE: If Ceph is being used, `ceph_poll.sh` should exit after several seconds and Ceph should be the default StorageClass. 
 
 
 To correct this issue:
-1. Uninstall Rook/Ceph: `./scripts/rmrook.sh`
-2. Uninstall Kubeflow: `./scripts/k8s_deploy_kubeflow.sh -D`
-3. Re-install Rook/ceph: `./scripts/k8s_deploy_rook.sh`
-4. Poll for Ceph to initialize (wait for this script to exit): `./scripts/ceph_poll.sh`
-5. Re-install Kubeflow: `./scripts/k8s_deploy_kubeflow.sh`
+1. Uninstall Rook/Ceph: `./scripts/k8s/rmrook.sh`
```

Comment:
Should we rename this to remove_rook.sh or delete_rook.sh to start a better naming convention? Better yet, can we modify the deploy_rook.sh script to have a `-d` flag like we've done elsewhere?

Reply:
renamed. will leave the flag for another PR